guides/doc-Managing_Configurations_Puppet/docinfo.xml

```diff
@@ -0,0 +1,14 @@
+<title>Managing Configurations Using Puppet Integration in Satellite</title>
+<productname>Red Hat Satellite</productname>
+<productnumber>7.0</productnumber>
```

Comment:
Probably unrelated, but in a lot of docinfo there's still 6.10. It would be good if those were all bumped to 7.0, both here and in 3.1.